src/EthMultiVault.sol

```diff
@@ -2072,17 +2239,16 @@ contract EthMultiVault is IEthMultiVault, Initializable, ReentrancyGuardUpgradea
     /// @return hash the corresponding hash for the given RDF triple
     /// @dev only applies to triple vault IDs as input
     function tripleHash(uint256 id) public view returns (bytes32) {
-        uint256[3] memory atomIds;
-        (atomIds[0], atomIds[1], atomIds[2]) = getTripleAtoms(id);
-        return keccak256(abi.encodePacked(atomIds[0], atomIds[1], atomIds[2]));
+        (uint256 subjectId, uint256 predicateId, uint256 objectId) = getTripleAtoms(id);
```

Comment:
3.16 - Utilize Helper Functions More

Here, we make better use of the helper methods as a single source of truth.